### ISSUE TITLE
indexing not triggered when external drive connected

### ISSUE THREAD

**Issue by mrawls** [nightlightapp/Nightlight-Issues#4]
When I plugged in an external USB hard drive, Nightlight did not find any FITS files on it. I can navigate to new directories in the Sources pane, but none of the directories on the new drive have been indexed. If there is a way to manually tell Nightlight to index it, I can't find it. This seems like a critical issue that should be resolved for v1.0 since lots of people have FITS files on multiple drives.

**Comment by demitri** [comment]
Is the drive formatted as a Mac volume, i.e. Mac OS Extended? You can check this by selecting the drive in the Finder, selecting "Get Info", and looking under "format". Spotlight doesn't index non-Mac drives (e.g. FAT, NTFS). More extensive file support is on the list, but I'm not sure what that will look like yet.

**Comment by mrawls** [comment]
Yes, it says it is Mac OS Extended (Journaled). I recently reformatted it because it was Windows-ified before. I tried launching Nightlight after the drive was plugged in and I also tried launching Nightlight before the drive was plugged in. Here is what a directory on the drive which I added to the Source pane looks like.
![nightlight_emptydir](https://cloud.githubusercontent.com/assets/5659077/14860517/1616bb16-0c5e-11e6-950b-14b4163b02ce.png)
